satpy/etc/readers/viirs_sdr.yaml

```diff
@@ -14,6 +14,7 @@ datasets:
     file_key: All_Data/{dataset_group}_All/Longitude
     units: "degrees"
```

Comment:
This should be degrees_east, and latitudes should be degrees_north

Reply:
Fixed.